rclcpp/test/rclcpp/test_executor.cpp

```diff
@@ -122,14 +122,14 @@ TEST_F(TestExecutor, constructor_bad_guard_condition_init) {
   auto mock = mocking_utils::patch_and_return(
     "lib:rclcpp", rcl_guard_condition_init, RCL_RET_ERROR);
   EXPECT_THROW(
-    std::make_unique<DummyExecutor>(),
+    std::unique_ptr<DummyExecutor>(new DummyExecutor()),
```

Comment:
You could have just done `auto throw_away = ...` instead. In fact this can be dangerous too, since make_unique (and make_shared) are carefully crafted to be exception safe.

But in this instance it's probably fine to just use `new`.

Reply:
Or maybe `(void)` before it would be fine.

Reply:
Should do it in a more C++ way:
```
static_cast<void>(std::make_unique<DummyExecutor>());
```
(Taken from https://stackoverflow.com/a/53581769)

Reply:
Changed now.